docs/UserGuide.md

```diff
@@ -189,91 +190,105 @@ All planned enhancements will also be listed in the [Planned Enhancements / Know
 
 ###  Add alternative information to contact: `addalt`
 
-Adds alternative contact information to an existing contact.
+Besides adding basic information for your contacts, CampusConnect supports the addition of alternative information of your existing contacts in CampusConnect. It helps you to store more contact information so that you can stay better connected to them.
 
-Format: `addalt INDEX [tg/TELEGRAM] [e2/SECONDARY_EMAIL] [li/LINKEDIN] [b/BIRTHDAY]`
+This feature involves the command: `addalt`, which adds alternative information of your contact.
 
-<box type="info">
-At least one of the optional fields must be provided. The fields you enter should follow the following format:
+Format: `addalt PERSON_INDEX [tg/TELEGRAM] [e2/SECONDARY_EMAIL] [li/LINKEDIN] [b/BIRTHDAY]`
+
+<box type="info" icon=":fa-solid-magnifying-glass:">
+
+Below are some examples on how to use the command:
 
-| Field             | Format                                                                                                        | Example               |
-|-------------------|---------------------------------------------------------------------------------------------------------------|-----------------------|
-| `TELEGRAM`        | Start with the `@` symbol, no whitespace with a minimum length of 5 characters. Use `a-z`, `0-9` and `_` only | @johndoe              |
-| `SECONDARY_EMAIL` | Have the format of `local-part@domain`                                                                        | johndoe@hotmail.com   |
-| `LINKEDIN`        | Use `a-z`, `A-Z`, `0-9`, `_` and `-` only                                                                     | john-doe-b9a38128a    |
-| `BIRTHDAY`        | Have the format of `DD/MM`                                                                                    | 31/10                 |
+* `addalt 1 tg/@johndoe_123 e2/johndoe@hotmail.com li/john-doe-b9a38128a b/31/10`: Adds telegram "@johndoe_123", secondary email "johndoe@hotmail.com", linkedin "john-doe-b9a38128a" and birthday "31/10" for contact at index 1 of your contact list.
+* `addalt 2 e2/besty@hotmail.com tg/@betsycrowe li/besty-crowe-b7a15138b b/24/07`: Adds secondary email "besty@hotmail.com", telegram "@bestycrowe", linkedin "besty-crowe-b7a15138b" and birthday "24/07" for contact at index 2 of your contact list.
+* `addalt 3 tg/janedoe_123`: Adds telegram "@janedoe_123" for contact at index 3 of your contact list.                           
+
+Refer to [properties of contact](#properties-of-contact) on the accepted formats for the respective parameters.
 
 </box>
 
-Examples:
-* `addalt 1 tg/@johndoe e2/johndoe@hotmail.com`
-    * Adds John Doe's telegram "@johndoe" and secondary email "johndoe@hotmail.com"
-* `addalt 1 li/john-doe-b9a38128a`
-    * Adds John Doe's linkedin "john-doe-b9a38128a"
+<box type="warning">
+
+* You should provide a positive integer that is smaller than the number of contacts stored in CampusConnect for `PERSON_INDEX`.
```

Comment:
Should it be smaller than the number of contacts currently listed?

Reply:
nice catch, the index definitely could be the same as the number of contacts! fixed in b33ffa82